README.md

```diff
@@ -1,5 +1,28 @@
 # Wikidata Reference Island
 
+## Installation and Running Tests
+We can create a python virtual environment.
+This has the advantage of keeping dependencies from this project separate from your system.
+
+Create a venv in a folder `./venv`
```

Comment:
Yes! I like the virtual env approach, I was reading about it early this morning, and even tried it, but it added lots of directories that I wasn't sure if we should track or not. What do you think, should we just set up a virtual env to be used by us all? (I made a right mess of python version in my system BTW...)

Also, it might be worthwhile to mention dependency management, still not sure how to install a package properly without handwriting it in the `requirments.txt` file.

Reply:
I don't think we should share the venv folder, it uses lots of binary files (wheels) that won't work in different systems. You can share the "pip freeze" though.